configure.ac

```diff
@@ -161,6 +161,8 @@ AS_IF([test "x$with_docs_only" = xyes],
      m4_include([config/m4/rccl.m4])
      m4_include([config/m4/sharp.m4])
      m4_include([config/m4/mpi.m4])
+     m4_include([config/m4/ibverbs.m4])
+     m4_include([config/m4/mlx5dv.m4])
```

Comment:
why do we have to separate config for ib and mlx5dv? I would expect ib and mlx5dv headers in the same install path